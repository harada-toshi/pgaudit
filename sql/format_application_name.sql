\! cp sql/format_application_name/format_application_name.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl -w restart -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! rm /tmp/config.conf
\! sh sql/format_application_name/format_application_name.sh

