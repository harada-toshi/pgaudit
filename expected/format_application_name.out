	        format = 'AUDIT:(1) %application_name,%d,%class,%command_text'
	        format = 'AUDIT:(2) %a,%d,%class,%command_text'
LOG:  AUDIT:(2) createdb,postgres,DDL,CREATE DATABASE foo;
LOG:  AUDIT:(1) psql,foo,READ,SELECT 1
LOG:  AUDIT:(2) dropdb,postgres,DDL,DROP DATABASE foo;
